.travis.yml

```diff
@@ -34,9 +54,10 @@ before_script:
       echo yes | pecl install apcu-5.1.6
     fi
   - |
-    if [[ $TRAVIS_PHP_VERSION != 'hhvm' ]]; then
+    if [[ $TRAVIS_PHP_VERSION != 'hhv*' ]]; then
```

Comment:
Don't think this will work, why even changing it (plain hhvm is used in above config)?